src/App.js

```diff
@@ -4,35 +4,34 @@ import { Switch, Route } from "react-router";
 // import { connect } from 'react-redux'
 // import { resizeWindow } from './actions'
 import "./App.css";
-import MapPage from "./components/MapPage";
-import Mission from "./components/pages/Mission";
-import Project from "./components/pages/Project";
-import Share from "./components/pages/Share";
-import Contribute from "./components/pages/Contribute";
-import Head from "./components/Head";
+import MapPage from "./components/MapPage/MapPage";
+import Mission from "./components/Pages/Mission";
```

Comment:
I'm getting this error when running locally: 
![image](https://user-images.githubusercontent.com/19523341/145303818-8740d46c-4721-4318-9bfc-eb26638a31af.png)

Seems like the directory is named `pages` but the import uses `Pages` -- resulting in this error. Changing it to `pages` resolves the error

Reply:
Fixed in https://github.com/phlask/phlask-map/pull/222/commits/17867d7015463627225ae851af75c5d0e5970dc1